docs/specs/03-skills-and-execution.md

```
@@ -91,59 +97,64 @@ Two-tier access:
 Explicitly listed in agent config (`pinned_skills`). Always available to the agent, always included in the LLM's tool list.
 
 ### Discoverable Skills
-All registered skills (local + MCP) are listed in a **skill registry**. A built-in `skill-registry` skill is available to all agents with `allow_discovery: true`. When an agent's LLM determines it needs a capability not in its pinned skills, it invokes:
+All registered skills (local + MCP) are searchable via the built-in `skill-registry` skill. Agents with `allow_discovery: true` in their YAML automatically receive `skill-registry` in their tool list. When the LLM determines it needs a capability not in its pinned skills, it invokes:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)